appinsights/metadata-v1.27.0.json

```
@@ -0,0 +1,15 @@
+{
+    "title": "Application Insights Metrics",
+    "metricCategory": "__MetricsCollection_appinsights__",
```

Please add this category to all metrics in the collection. P.S. It can use capitalization, so we can do:

```suggestion
    "metricCategory": "__MetricsCollection_AppInsights__",
```

---

Updated

---

Just found that category have a maximum length of 30. Should we increase this limit here? @risinh

---

I am a bit confused on "add this category to all metrics" part, since previous metric collection also assign a short value to categories (like "GenAI") in metric level as well, I will keep as is, but will update in future PRs if you have further suggestions